hvm.sh

```diff
@@ -4,13 +4,15 @@ source $HVM/versions.sh
 source $HVM/platform.sh
 
 hvm_get_haxe_versions() {
+	PREVIEW="\(-\(alpha\|beta\|rc\)[0-9]\)\?"
+	VERSION="[^-]*${PREVIEW}"
 	case $PLATFORM in
-		'OSX') local DOWNLOAD='href="haxe-[^-]*-osx.tar\.gz' ;;
-		'LINUX32') local DOWNLOAD='href="haxe-[^-]*-linux\(32\)\?.tar\.gz' ;;
-		'LINUX64') local DOWNLOAD='href="haxe-[^-]*-linux64.tar\.gz' ;;
+		'OSX') local DOWNLOAD="href='haxe-${VERSION}-osx.tar\.gz'" ;;
```

Comment:
does changing `"` to `'` work here? I would think it would need to be a literal match for the html on the page, which is currently `"`...